### PR TITLE
Leaderboard Setup

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In this pull request:
- I set up Gitflow for the project
- I set up Webpack for the project
- I used  HTML markup to make the layout

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <div class="add">
+        <h2>Add your score</h2>
+        <form>
+          <input type="text" placeholder="Your name" />
+          <input type="number" placeholder="Your score" />
+          <button type="submit" class="score-submit">submit</button>
+        </form>
+      </div>
+    </div>
+  </body>
```
Comment (Munsa1) [inline dist/index.html:35]: - [ ] Kindly position the elements as in the wireframe provided, styling is NOT required, BUT placement of the elements should be followed as per requirement

Provided
image

Given
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @Nyame-Wolf 👋,

_This is an additional review as a result of the second opinion request._

I confirm that the app has the basic wireframe styling. I think your project can be approved.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by OLIPLICHE — APPROVED** [review]
Hi @Nyame-Wolf , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**